### PR TITLE
Challenge 05

### PR DESCRIPTION
**Pull request by amirebrahimi**
This is courtesy of @Shiro-Raven.

Please make any final suggestions/edits before 6pm PST when I will make it live.

### CODE REVIEW

**Comment by amirebrahimi** [comment]
Closes #23

File: challenge-05/qosf-monthly-challenge-05.ipynb [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    "For a more lengthy and comprehensive intro, you can check resource [A]."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Why QML though?\n",
+    "\n",
+    "Even though classical ML and the so-called deep neural networks have contributed significantly to many disciplines they are not perfect. Even state-of-the-art algorithms may require a significant amount of training time given a huge volume of data present. While QML also would be affected by a huge volume of data, one potential application of QML is to improve training by leveraging quantum phenomenon. If the data happens to be quantum in nature, then QML can shine if run on quantum computers where it might be expensive to simulate on classical computers.\n",
```
Comment (Shiro-Raven) [inline challenge-05/qosf-monthly-challenge-05.ipynb:44]: I think "phenomena" makes more sense than "phenomenon" here

File: challenge-05/qosf-monthly-challenge-05.ipynb [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    "\n",
+    "2. Now that you got a feel of implementing a QML model, let's go for something more challenging. The [MNIST dataset](https://www.kaggle.com/c/digit-recognizer/data) is another benchmark dataset that is typically used to check the sanity of Computer Vision models. Dealing with this dataset is tricky for two reasons: it is highly-dimensional (28 x 28 images = 784 components!), and it contains way more classes than the Iris dataset. Think about what you can do to use as few qubits as possible (Hint: check resource [C]). As for a higher number of classes, it is okay to start with only two arbitrary classes, excluding all the others, and adding more classes as you feel comfortable with your approach.  \n",
```
Comment (Shiro-Raven) [inline challenge-05/qosf-monthly-challenge-05.ipynb:70]: A suggestion for a more helpful hint

```suggestion
    "2. Now that you got a feel of implementing a QML model, let's go for something more challenging. The [MNIST dataset](https://www.kaggle.com/c/digit-recognizer/data) is another benchmark dataset that is typically used to check the sanity of Computer Vision models. Dealing with this dataset is tricky for two reasons: it is highly-dimensional (28 x 28 images = 784 components!), and it contains way more classes than the Iris dataset. Think about what you can do to use as few qubits as possible (Hint: think of it as using as few "dimensions" as possible. If you give up, and can't think of a way, check resource [C]). As for a higher number of classes, it is okay to start with only two arbitrary classes, excluding all the others, and adding more classes as you feel comfortable with your approach.  \n",
```

File: challenge-05/qosf-monthly-challenge-05.ipynb [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    "\n",
+    "2. Now that you got a feel of implementing a QML model, let's go for something more challenging. The [MNIST dataset](https://www.kaggle.com/c/digit-recognizer/data) is another benchmark dataset that is typically used to check the sanity of Computer Vision models. Dealing with this dataset is tricky for two reasons: it is highly-dimensional (28 x 28 images = 784 components!), and it contains way more classes than the Iris dataset. Think about what you can do to use as few qubits as possible (Hint: think of it as using as few "dimensions" as possible. If you give up, and can't think of a way, check resource [C]). As for a higher number of classes, it is okay to start with only two arbitrary classes, excluding all the others, and adding more classes as you feel comfortable with your approach.  \n",
```
Comment (Shiro-Raven) [inline challenge-05/qosf-monthly-challenge-05.ipynb:70]: There is a typo here

```suggestion
    "2. Now that you got a feel of implementing a QML model, let's go for something more challenging. The [MNIST dataset](https://www.kaggle.com/c/digit-recognizer/data) is another benchmark dataset that is typically used to check the sanity of Computer Vision models. Dealing with this dataset is tricky for two reasons: it is highly-dimensional (28 x 28 pixels = 784 dimensions!), and it contains way more classes than the Iris dataset. Think about what you can do to use as few qubits as possible (Hint: think of it as using as few "dimensions" as possible. If you give up, and can't think of a way, check resource [C]). As for a higher number of classes, it is okay to start with only two arbitrary classes, excluding all the others, and adding more classes as you feel comfortable with your approach.  \n",
```

**Pull request merged**